pimcore/models/Object/ClassDefinition/Data/Classificationstore.php

```diff
@@ -1186,6 +1191,8 @@ public function getAllowedGroupIds()
      */
     public function setAllowedGroupIds($allowedGroupIds)
     {
+        $parts = [];
```

Comment:
Not sure which implications this has as groups were set to null before, now to an empty array. Please add a `TODO` to check this, but do not alter the behaviour.